lib/mercator.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * @typedef {Object} Point
+ * @property {Number} x
+ * @property {Number} y
+ */
+
+/**
+ * @typedef {Object} LatLng
+ * @property {Number} lat
+ * @property {Number} lng
+ */
+
+/**
+ * @typedef {Object} Tile
+ * @property {Number} x
+ * @property {Number} y
+ * @property {Number} z
+ */
+
+/**
+ * @typedef {Object} TileBounds
+ * @property {LatLng} sw
+ * @property {LatLng} ne
+ */
+
+/**
+ * @typedef {Object} Pixel
+ * @property {Number} x
+ * @property {Number} y
+ */
+
+/**
+ * @typedef {Object} Evt
+ * @property {Pixel} pixel
+ * @property {LatLng} latLng
+ */
```

Comment:
If you can, either log this out in use by one of the examples or catch it in the debugger. I'll bet it's the same thing as the `google.maps.MapMouseEvent`.